test/modules/geo-layers/mvt-layer.spec.js

```diff
@@ -145,12 +145,12 @@ test('TileJSON', async t => {
   };
 
   const testCases = [
-    {
-      props: {
-        data: 'http://echo.jsontest.com/key/value'
-      },
-      onAfterUpdate
-    },
+    // {
```

Comment:
After the update to JSONLoaders, it throws the following error: 

```Error: An error occurred fetching TileJSON: Error: No valid loader found data: "", contentType: ""```

So, it looks we need to mock fetch to make it works with loaders.gl, but it's not easy because loaders.gl wait for a Response object which is not available in NodeJS. Something like this:

```js
_global.fetch = url => {

    const headers = {
      status: 200,
      statusText: 'OK',
      'Content-Type': 'application/json'
    };
    const r = new Response(JSON.stringify(tileJSON), headers);

    return Promise.resolve(r);
  };
```

So... after all of this, I think we're overcomplicating the test and it doesn't bring too much value, so I just decided to remove it.

Reply:
Import ‘@loaders.gl/polyfills’ to get fetch / Response etc in node